hello.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Println("Hello, World!")
+}
```

Comment:
Run `gofmt -w .` in the current directory of a folder/file to format all Go code contained in that folder.
For this line for instance, it's supposed to end with a new line.
It's opinionated, I know :)